File: app/src/main/java/com/neocaptainnemo/cv/model/ProjectsResponse.java
package com.neocaptainnemo.cv.model;

import com.google.firebase.database.IgnoreExtraProperties;
import com.google.firebase.database.PropertyName;

import java.util.List;
import java.util.Map;

@IgnoreExtraProperties
public class ProjectsResponse {

    @PropertyName("projects")
    public List<Project> projects;

    @PropertyName("resources")
    public Map<String, Map<String, String>> resources;

}

File: app/src/main/java/com/neocaptainnemo/cv/model/ProjectsResponse.kt
package com.neocaptainnemo.cv.model

import com.google.firebase.database.IgnoreExtraProperties
import com.google.firebase.database.PropertyName

@IgnoreExtraProperties
class
ProjectsResponse {

    @PropertyName("projects")
    var projects: ArrayList<Project> = arrayListOf()

    @PropertyName("resources")
    var resources: HashMap<String, Map<String, String>> = hashMapOf()

}
